app/javascript/react/components/ShopReviewsList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import ShopReviewTile from "./ShopReviewTile"
+
+const ShopReviewsList = (props) => {
+  const reviewTiles = props.reviews.map((review) => {
+    return (
+      <ShopReviewTile review={review} />
+    )
+  })
+
+  return(
+    <p>{reviewTiles}</p>
```

Comment:
your review tiles are a collection of `div` elements, and it's technically invalid syntax to have a `div` inside of a `p` paragraph. Alternatively we can use another `div` here or a fragment `<>`